frontend/src/resources/hooks/useFirstRenderEffect.tsx

```diff
@@ -0,0 +1,11 @@
+import { useEffect, useRef } from 'react';
+
+export const useFirstRenderEffect = (effectFn: () => void) => {
```

Comment:
This seems over-complicated over simply calling `useEffect` - is there any difference between the two?